source/main.cpp

```diff
@@ -1,14 +1,24 @@
+#include "PrintDiamond.h"
+
 #include <iostream>
 #include <string>
 
 using namespace ::std;
 
 int main()
 {
-	string s;
+	PrintDiamondClass PDMain;
+
+	cin >> PDMain.m_input_char;
 
-	cout << "Hello World!";
+	PDMain.InitArray();
+	PDMain.LetterToNum();
+	PDMain.PrintDiamond();
+	PDMain.ReversePrintDiamond();
+	PDMain.PrintGrid();
```

Comment:
All these functions must be called to produce a diamond, and in this specific order. To make it easy on the user of your class, you should make them all private and add a single public function to call them in this order.